nuxt.config.js

```diff
@@ -101,9 +101,15 @@ export default {
 	*/
 	build: {
 		/*
-		** You can extend webpack config here
-		*/
-		extend (config, ctx) {
+				** You can extend webpack config here
```

Comment:
Please fix the indentations on line 104-105. Interesting that the linting script doesn't report error on it.

BTW, the origin of the `config.node` solution for `fs and child_process not found` error comes from [here](https://github.com/nuxt-community/dotenv-module/issues/11#issuecomment-376780588). You can add the link to the code comment that @greatislander suggested to add, which makes sense.